docs/components/DocumentationSidebar.tsx

```diff
@@ -122,7 +122,7 @@ export default class DocumentationSidebar extends React.Component<Props> {
     };
 
     return (
-      <nav css={STYLES_SIDEBAR} {...customDataAttributes}>
+      <nav css={STYLES_SIDEBAR} {...customDataAttributes} aria-label="The basics">
```

Comment:
I'm not sure if we should hardcode this aria-label here, might need to add it to `navigation.js` (but that's pretty hard atm 😢 )

Reply:
I've switched to something more generic: "Table of contents", since the left navbar is mostly about presenting sections and links under each section, no matter the section 😊 .

Let me know if it does not help to fix the scoping issue of "The basics" and if it does not, I'll be more than happy to dig into the navigation thing with some guidance 🙏🏻 

😊